### PR TITLE
feat: adding bbc_radio_one_relax to LiveBroadcastHelper and removing from deny list (to make it appear in /schedules)

### PR DESCRIPTION
**Pull request by spacekitcat**
https://jira.dev.bbc.co.uk/browse/DATCAP-165

I've removed p093xs25 from the deny list. I'm unsure if this could have any unintended consequences. 

Another other thing that concerns me is the display name on the schedules homepage:

image

That doesn't look right.

### CODE REVIEW

**Comment by ajroberts10** [comment]
Looks good otherwise 👍🏻  The name in the listing is driven from the database. This is controlled by editorial I think so they'll probably update it prior to go live (or it has an updated value on TEST / LIVE) 

Screenshot 2021-02-24 at 17 58 56

File: src/Controller/Schedules/HomeController.php [hunk 1]
```diff
@@ -23,7 +23,6 @@ class HomeController extends BaseController
         'p02yxy62' => true, // BBC World Service US Public Radio
         'p00hwfhp' => true, // BBC WORLD NEWS Americas
         'p00fzlgd' => true, // Radio 10
-        'p093xs25' => true, // Radio 1 extension
```
[the remark this answers is not in the document]
Comment (spacekitcat) [reply src/Controller/Schedules/HomeController.php:26]: But then it doesn't display in `/schedule`, there's a check in the main for loop that iterates over the service list.

[the remark this answers is not in the document]
Comment (spacekitcat) [reply src/Controller/Schedules/HomeController.php:26]: As far as I can ascertain, this deny list is for programmes that don't have any concept of a live broadcast along with some other edge cases.

[the remark this answers is not in the document]
Comment (spacekitcat) [reply src/Controller/Schedules/HomeController.php:26]: "    // Hide unwanted services and non-integrated language stations from listings "

[the remark this answers is not in the document]
Comment (spacekitcat) [reply src/Controller/Schedules/HomeController.php:26]: I don't know, it would be good to understand the business logic this code describes.

**Comment by spacekitcat** [comment]
I saw that in the network table. I was wondering if it should be corrected before merging.

File: src/Controller/Schedules/HomeController.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (ajroberts10) [reply src/Controller/Schedules/HomeController.php:26]: I think the pid of the brand we're adding is `p093xvct` so we should be ok to leave this one on this list

**Review by murkas — APPROVED** [review]
This all looks good, we have established that the blacklisted service was removed earlier in the year. Good find, Lisa!

**Review by ajroberts10 — APPROVED** [review]

**Comment by ajroberts10** [comment]
We spoke to Jatin about this particular zid in the deny list and it appears it was hidden intentionally until now. We're good to remove this zid from the deny list 👍🏻  Good find!

File: src/Controller/Schedules/HomeController.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (spacekitcat) [reply src/Controller/Schedules/HomeController.php:26]: Wait, where did p093xvct come from?

**Pull request merged**